### ISSUE TITLE
[BUG] Cannot click on items in Outline pane

### ISSUE THREAD

**Issue by gautams3** [JunoLab/Juno.jl#484]
## Details
- Atom version: 1.44.0 x64
- Julia version: 1.3.1
- OS: macOS 10.13
- Electron: 4.2.7
**Thrown From**: [julia-client](https://github.com/JunoLab/atom-julia-client) package 0.12.0

- Package versions:
  - Atom.jl: 0.11.3
  - julia-client: 0.12.0
  - ink: 0.12.0

### Steps to reproduce

1.  Create foo.jl
```
function foo(x)
  return x + 5
end

foo(3)
```
2. Wait till the Outline pane gets the outline of the file. I don't know how to force this to update so I just wait.
3. Once updated, you should see `λ foo(x)` in the outline pane. Click on it.
4. Observe the stack trace below, thrown from the `julia-client` package.

### Stack Trace

Uncaught TypeError: Invalid Point: (NaN, 0)

```
At /Applications/Atom.app/Contents/Resources/app/static/<embedded>:11

TypeError: Invalid Point: (NaN, 0)
    at Function.Point.assertValid (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:14:49670)
    at TextBuffer.clipPosition (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:496566)
    at MarkerLayer.setMarkerRange (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:14:81851)
    at Marker.update (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:14:72274)
    at Marker.setHeadPosition (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:14:68833)
    at DisplayMarker.setHeadBufferPosition (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:14:2911860)
    at changePosition (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:522509)
    at Cursor.changePosition (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:530896)
    at Cursor.setBufferPosition (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:522475)
    at moveCursors.o (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:29229)
    at Array.forEach (<anonymous>)
    at transact (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:33144)
    at TextBuffer.transact (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:492268)
    at TextEditor.transact (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:24215)
    at TextEditor.moveCursors (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:33112)
    at TextEditor.setCursorBufferPosition (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:29212)
    at Outline.item.onClick (/packages/julia-client/lib/runtime/outline.js:102:14)
```

### Commands

```
  2x -5:03.8.0 core:save (input.hidden-input)
     -3:31.7.0 core:move-left (input.hidden-input)
     -3:31.3.0 core:backspace (input.hidden-input)
     -3:30.6.0 core:save (input.hidden-input)
     -3:01.2.0 editor:delete-to-beginning-of-word (input.hidden-input)
     -3:00.9.0 core:confirm (input.hidden-input)
     -2:58.4.0 core:close (input.hidden-input)
  2x -0:36.7.0 core:move-down (input.hidden-input)
     -0:35.2.0 core:save (input.hidden-input)
     -0:34.3.0 editor:newline (input.hidden-input)
     -0:33.9.0 core:save (input.hidden-input)
     -0:29.8.0 inline:clear-current (input.hidden-input)
     -0:29.8.0 editor:consolidate-selections (input.hidden-input)
     -0:29.8.0 core:cancel (input.hidden-input)
     -0:24 core:select-all (input.hidden-input)
  2x -0:23 core:copy (input.hidden-input)
```

**Comment by gautams3** [comment]
One workaround is to install older versions of `ink` and `julia-client`

```
> apm unininstall ink
> apm unininstall julia-client
> apm ininstall ink@0.11.8
> apm ininstall julia-client@0.11.3
```

**Comment by aviatesk** [comment]
update Atom.jl to 0.12.2. you should've got a notification about this if you're on julia-client@0.12.0

**Comment by pfitzseb** [comment]
> you should've got a notification about this if you're on julia-client@0.12.0

So far I don't think anyone has reported seeing that warning pop up...

**Comment by aviatesk** [comment]
So mysterious, it's only visible to us 😱 
![image](https://user-images.githubusercontent.com/40514306/74738479-54d66780-529a-11ea-811e-81ddf935a325.png)

**Comment by gautams3** [comment]
I think I had to upgrade (`ink` and `julia-client`), close my current project folder, and then create a new file after which I saw this warning. It definitely didn't come up immediately after upgrading the packages.

**Comment by aviatesk** [comment]
ah so you mean you saw this warning (when you booted Julia) ?
Do you think this wasn't informativ e enough to encourage you to update `Atom` ?

**Comment by gautams3** [comment]
I had the chance to reproduce this problem on Ubuntu 18.04. Here's what I did

1. Updated `ink` (0.11.8 -> 0.12.0) and `julia-client` (0.11.3 -> 0.12.0)
2. Restart Atom (as instructed by the package update)
3. Start julia in the REPL by pressing `Enter`
4. No warning shows up. I checked that `Atom.jl` is `v0.11.3` (via `]st` in REPL)
5. Opened a new window (closed previous project window). New `foo.jl` file on Desktop (with the same `foo(x)` in my description above). New Julia instance. Still no warning.
6. Waited for the outline to load. I click on `λ foo(x)`. Get the same error and stack trace.
7. Restart Atom. Still no warning like the one above.

I don't know how to get the warning notification to show up at this point.
